### DISCUSSION TITLE
Stand ground?

### DISCUSSION THREAD

**Discussion by Chonkblonk** [diasurgical/DevilutionX#6903] [Q&A] [answered]
I can find the entry for binding Stand Ground for the controller, both in the menu and the .ini file, but I can't seem to find where I can change that bind otherwise? I would like to bind it to a mouse button.

**Comment by DakkJaniels** [comment]
Currently not possible.

**Reply by Chonkblonk** [reply]
Oh. Well, I hope that's added in some update some day, it would be more comfortable for me on lazy days when I lay back in a couch or bed with my laptop. As is I can basically play Diablo with just the mouse when playing Sorcerer.

Actually, if I turn on my gamepad thing, I could just use a single button on that for holding ground, right? Even if I otherwise use the mouse for almost everything else.

**Reply by StephenCWills** [reply, marked as answer]
> Actually, if I turn on my gamepad thing, I could just use a single button on that for holding ground, right? Even if I otherwise use the mouse for almost everything else.

Unfortunately, I don't think it works that way. The logic for mouse-and-keyboard checks the "mod state" of the keyboard directly to determine whether Shift/Ctrl are pressed, which is a large part of why the binding can't be changed. The gamepad implementation of Stand Ground works for the "primary action button" and d-pad movement which is triggered by both gamepad and touchscreen controls, but not by mouse-and-keyboard.

**Reply by Chonkblonk** [reply]
Unfortunate. I guess I can only hope that you guys will add the option for some future update.
